pkg/microservice/aslan/core/service/service/types.go

```diff
@@ -84,6 +85,12 @@ type CreateFromChartTemplate struct {
 	Variables    []*Variable `json:"variables"`
 }
 
+type CreateFromChartRepo struct {
+	ChartRepoName string `json:"chartRepoName"`
```

Comment:
What about using `chart_repo_name` in json tag?

Reply:
need to keep same style with other attribute in one http request